Joining together CTRP data.Rmd

```diff
@@ -0,0 +1,304 @@
+AUC_2 <- fread("AUC_2.csv")
+experiment_info <- fread("Experiment_info.csv")
+cancer_cell_line <- fread("cancer_cell_line_info.csv")
+compound <- fread("compound_info.csv")
+  
+```
+
+## 3. Tidy the data frames
+
+### 3a. AUC_1
+```{r AUC_1}
+
+# Rename the columns in AUC_1 file
+
+AUC_1_rename <- AUC_1 %>%
+    rename("Exp._ID" = experiment_id, "AUC" = area_under_curve, "Compound_ID" = master_cpd_id) %>%
+    select("Exp._ID", "Compound_ID", "AUC")
+
+table1 <- head(AUC_1_rename)
+kable(table1, align = "ccc")
+
+```
+
+
+### 3b. AUC_2
+```{r AUC_2}
+
+# Rename the columns in AUC_2 file
+
+AUC_2_rename <- AUC_2 %>%
+    rename("Exp._ID" = experiment_id, "AUC" = area_under_curve, "Compound_ID" = master_cpd_id) %>%
+    select("Exp._ID", "Compound_ID", "AUC")
+
+table2 <- head(AUC_2_rename)
+kable(table2, align = "ccc")
+
+```
+
+### 3c. Experiment info
+```{r Experiment info}
+
+# Rename the columns in experiment_info file
+
+experiment_info_rename <- experiment_info %>%
+    rename("Exp._ID" = expt_id, "Date" = experiment_date, "Cells_per_well" = cells_per_well, "Cell_line_ID" = master_ccl_id) %>%
+    mutate(Date = ymd(Date)) %>%
+    select("Exp._ID", "Date", "Cell_line_ID", "Cells_per_well")
+
+table3 <- head(experiment_info_rename)
+kable(table3, align = "cccc")
+
+```
+
+### 3d. Cancer cell line info
+```{r Cancer cell line info}
+
+# Rename the columns in cancer_cell_line_info file
+
+cancer_cell_line_rename <- cancer_cell_line %>%
+    rename("Cell_line_name" = ccl_name, "Cancer_type" = cancer_type, "Cell_line_ID" = master_ccl_id)
+
+table4 <- head(cancer_cell_line_rename)
+kable(table4, align = "cll")
+
+```
+
+### 3e. Compound info
+```{r Compound info}
+
+# Rename the columns in compound_info file
+
+compound_rename <- compound %>%
+    rename("Compound_ID" = master_cpd_id, "Compound_name" = cpd_name, "Target_gene" = gene_symbol_of_protein_target)
+
+table5 <- head(compound_rename)
+kable(table5, align = "clc")
+
+```
+### 3f. Combine the data frames
+```{r Combining data frames}
+
+# Combine the AUC_1 and AUC_2 data frames into one - AUC_info
+
+AUC_info <- rbind(AUC_1_rename, AUC_2_rename)
+
+# Combine experiment_info with cancer_cell_line_info 
+
+experiment_cancer_cell_line <- experiment_info_rename %>%
+      full_join(cancer_cell_line_rename, by = "Cell_line_ID")
+
+# Combine AUC_info and experiment_cancer_cell_line
+
+AUC_exp_cancer <- AUC_info %>%
+      full_join(experiment_cancer_cell_line, by = "Exp._ID")
+
+# Add compound_info to experiment_cancer_cell_line
+
+AUC_exp_cancer_compound <- AUC_exp_cancer  %>%
+     full_join(compound_rename, by = "Compound_ID")
+
+# Rename the column in the combined data frame
+
+all_data <- AUC_exp_cancer_compound %>%
+    rename("Cells/well" = "Cells_per_well")
+
+# Rorder the columns in the combined data frame
+
+all_data<- select(all_data, "Date", "Exp._ID", "Cell_line_name", "Cell_line_ID", "Cancer_type", "Cells/well", "Compound_name", "Compound_ID", "AUC", "Target_gene")
```

Comment:
Altogether, nice job combining the different data frames into one. I also like the use of select() to reorganize the columns at the end. I didn't reorganize the columns and the table was pretty messy. This is much nicer!